src/raiden_libs/matrix.py

```diff
@@ -225,12 +229,12 @@ def follow_address_presence(self, address: Address, refresh: bool = False) -> No
 
     def _get_user_from_user_id(self, user_id: str) -> User:
         """Creates an User from an user_id, if none, or fetch a cached User """
-        for broadcast_room in self.broadcast_rooms:
-            if user_id in broadcast_room._members:  # pylint: disable=protected-access
-                user: User = broadcast_room._members[user_id]  # pylint: disable=protected-access
-                break
+        if (  # pylint: disable=protected-access
```

Comment:
It's a better idea to not make `_broadcast_room` a private member.

Reply:
Also, shouldn't this be an `assert self._broadcast_room`?

Reply:
> It's a better idea to not make `_broadcast_room` a private member.

I think this is about accessing `Room._members`, not `_broadcast_room`.

Reply:
Regarding the assert, I changed it now. But I'm not sure about it.

Reply:
@ulope Can you throw a look at this and check whether it makes sense?

Reply:
Yeah looks ok.

Maybe it would make that you also start using the `DisplayNameCache`, although using the broacasting room members as the "cache" should already give you 90% of the benefit.